content/pension_pot_options.md

```diff
@@ -202,16 +295,15 @@ You can mix your options at different times in your retirement. For example, you
 
 Pension providers are developing new retirement income products all the time, some of which might mix these options.
 
-$E
```

Comment:
I'll update the example to this - good spot.

On 21 April 2015 at 09:01, Andrew Garner <notifications@github.com> wrote:

> In content/pension_pot_options.md
> <https://github.com/guidance-guarantee-programme/pension_guidance/pull/161#discussion_r28757060>
> :
>
> > @@ -202,16 +295,15 @@ You can mix your options at different times in your retirement. For example, you
> >
> >  Pension providers are developing new retirement income products all the time, some of which might mix these options.
> >
> > -$E
>
> You can avoid having to fall back to HTML if you use Kramdown's inline
> attribute lists:
>
> {: .example--mix }
> $E**Example**
>
> Your pension pot is £60,000. You decide to take 25% as tax-free cash which is £15,000.
>
> {: .example__annuity}
> You use £20,000 to buy an annuity. This gives you a taxable income for the rest of your life of about £900 a year.
>
> {: .example__flexible}
> You also put £25,000 into a drawdown fund and take out about £5,000 a year for about 5 years.
> $E
>
> The {: .foo} specifies attributes to add to the preceding element. So in
> this case we're adding a class of example--mix to the example block.
> Later we're adding classes of example__annuity and example__flexible to
> the paragraphs.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guidance-guarantee-programme/pension_guidance/pull/161/files#r28757060>
> .
>